lib/developer_portal/config/initializers/liquid.rb

```diff
@@ -22,7 +22,6 @@
 
   [
     Liquid::Tags::IncludeWithComments,
-    Liquid::Tags::AuthorizeNetForm,
```

Comment:
Do we have to leave this here? 🤔 `Liquid::Tags::OgoneForm` is still there...

Reply:
idk about this

Reply:
I think we should keep it to prevent breaking the portals with existing tags.